lib/esignatur/pades.rb

```diff
@@ -8,11 +8,12 @@ module Esignatur
   class Pades
     include ApiResource
 
-    attr_reader :attributes, :order
+    attr_reader :attributes, :order, :document
 
-    def initialize(order:, api:)
+    def initialize(order:, document:, api:)
```

Comment:
document is same as `@attributes`, so I would suggest to use only one field:

```suggestion
    def initialize(order:, attributes: {}, api:)
```